cpp/scenario/gazebo/src/utils.cpp

```diff
@@ -170,16 +170,17 @@ std::string utils::getWorldNameFromSdf(const std::string& fileName,
 
 std::string utils::getEmptyWorld()
 {
+    // The empty world matches default.sdf from upstream without models
     const std::string world = R""""(<?xml version="1.0" ?>
 <sdf version="1.6">
     <world name="default">
-        <physics default="true" type="dart">
+        <physics default="true" type="ignored">
```

Comment:
This is not a spec compliant SDF, but this is definitely not related to this PR, I opened https://github.com/osrf/sdformat/issues/550 for track the actual problem.

Reply:
Thanks! I think that currently Ignition supports selecting physics engine through the [`PhysicsEnginePlugin`](https://github.com/ignitionrobotics/ign-gazebo/blob/ign-gazebo5/include/ignition/gazebo/components/PhysicsEnginePlugin.hh) introduced in https://github.com/ignitionrobotics/ign-gazebo/pull/136 (couldn't find the actual PR). I already have a local branch that I'm about to push that introduces that component in our logic. I'm not really sure whether the `/world/physics@type` attribute is even read nor supported in Ignition.

Reply:
Btw, I'll proceed merging as it is. We can update it as soon as we figure this out, both current and new attributes are not breaking any existing logic.